src/cfnlint/data/ExtendedSpecs/all/03_value_types/aws_lambda.json

```diff
@@ -22,5 +22,63 @@
       "NumberMax": 10000,
       "NumberMin": 1
     }
+  },
+  {
+    "op": "add",
+    "path": "/ValueTypes/AWS::Lambda::EventSourceMapping.EventSourceArn",
+    "value": {
+      "GetAtt": {
+        "AWS::DynamoDB::Table": "StreamArn",
+        "AWS::Kinesis::Stream": "Arn",
+        "AWS::SQS::Queue": "Arn"
+      },
+      "Ref": {
+        "Parameters": [
+          "String"
+        ],
```

Comment:
Can you elaborate on this? Assuming this corresponds to [Parameter types](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html#parameters-section-structure-properties) based on what's been added so far, but not sure what the possible values are (since format is slightly different than CloudFormation's) or where this is used

```bash
$ git grep -h -A 2 'Parameters' src/cfnlint/data/ExtendedSpecs/all/03_value_types* | grep -v 'Parameters' | grep -v ']' | sed 's/ //g' | sort | uniq -c | sort -nr
  45 "String"
   6 "Strings"
   2 "SecurityGroupIds"
   2 "SecurityGroupId"
   2 "Number"
   1 "VpcId",
   1 "SubnetIds"
   1 "SubnetId"
   1 "KeyPair"
   1 "ImageId"
   1 "AvailabilityZones"
   1 "AvailabilityZone"
```

Reply:
It corresponds to [this](https://github.com/aws-cloudformation/cfn-python-lint/blob/master/src/cfnlint/data/ExtendedSpecs/all/02_parameter_types.json#L38).  Where we define the different types that can make up a String (for this example).